v4/Http/Controllers/SurveyController.php

```diff
@@ -0,0 +1,249 @@
+<?php
+
+namespace v4\Http\Controllers;
+use Ramsey\Uuid\Uuid;
+use Ushahidi\App\Validator\LegacyValidator;
+use v4\Models\Attribute;
+use v4\Models\Survey;
+use Illuminate\Http\Request;
+use Illuminate\Validation\Validator;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Validation\Rule;
+
+
+class SurveyController extends V4Controller
+{
+    protected static function getRules() {
```

Comment:
we don't _need_ a getRules function but I feel it's cleaner since we reuse the rules to have them here (rather than copy them). Just mentioning because it may seem like its an eloquent thing otherwise

Reply:
also errr I will move the rules to the model, makes no sense that they are in the controller :O